Scrabble.java

```diff
@@ -87,7 +109,16 @@ public static void playHand(String hand) {
 			String input = in.readString();
 			//// Replace the following break statement with code
 			//// that completes the hand playing loop
-			break;
+			if (isWordInDictionary(input)){
+				score+= wordScore(input);
+				hand = MyString.remove(hand, input);
+				System.out.println(input+" earned "+wordScore(input)+" points. Score: "+score+" points"+"\n");
+			}else if (!isWordInDictionary(input) && !input.equals(".")){
+				System.out.println("Invalid word. Try again.");

```

Comment:
this is incorrect.. 
case: 
1. hand : b a r n , dict = [barnicle], input= barnicle.
should not remove, print error.
your code does